services/circulation-service/src/main/resources/static/openapi/circulation-v1.json

```
@@ -45,10 +48,39 @@
+          { "$ref": "#/components/parameters/IfMatch" }
+        ],
+        "requestBody": {
+          "required": true,
+          "content": { "application/json": { "schema": { "$ref": "#/components/schemas/UpdateCirculationPolicyRequest" } } }
+        },
+        "responses": {
+          "200": { "$ref": "#/components/responses/PolicyUpdated" },
+          "400": { "$ref": "#/components/responses/BadRequest" },
+          "401": { "$ref": "#/components/responses/Unauthorized" },
+          "403": { "$ref": "#/components/responses/Forbidden" },
+          "429": { "$ref": "#/components/responses/TooManyRequests" },
```

Biome (2.5.6)

[error] The key 429 was already declared.

(lint/suspicious/noDuplicateObjectKeys)